crates/matrix-sdk-indexeddb/Cargo.toml

```diff
@@ -15,16 +15,17 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
 default = ["e2e-encryption"]
-e2e-encryption = ["matrix-sdk-base/e2e-encryption", "dep:matrix-sdk-crypto"]
+e2e-encryption = ["matrix-sdk-base/e2e-encryption", "dep:matrix-sdk-crypto", "dashmap"]
 experimental-timeline = ["matrix-sdk-base/experimental-timeline"]
 
 
 [dependencies]
 anyhow = "1.0.57"
 async-trait = "0.1.53"
 base64 = "0.13.0"
-dashmap = "5.2.0"
-futures-util = { version = "0.3.21", default-features = false }
+dashmap = { version = "5.2.0", optional = true }
+derive_builder = "0.11.2"
+futures = { version = "0.3.21", default-features = false, features = ["alloc"] }
```

Comment:
Let's please not do this. The futures crate isn't using the `crate?/feature` syntax (presumably because of MSRV), so activating the `alloc` feature is going to pull in basically all of the futures-* crates: https://github.com/rust-lang/futures-rs/blob/5c17408232e07fe6619ce7d501839ffdeab9c88f/futures/Cargo.toml#L38